core/frontend/src/RealityDataSourceTilesetUrlImpl.ts

```diff
@@ -115,7 +115,7 @@ export class RealityDataSourceTilesetUrlImpl implements RealityDataSource {
   }
 
   public getTileContentType(url: string): "tile" | "tileset" {
-    return url.endsWith("json") ? "tileset" : "tile";
+    return new URL(url, "https://localhost/").pathname.toLowerCase().endsWith("json") ? "tileset" : "tile";
```

Comment:
@mborjesson-bentley 
why is the base url here localhost and not something like `window.location.origin`?

Reply:
in this case we only look at pathname, so baseurl can be anything. the reason i chose localhost is because it's done the same way here:
https://github.com/iTwin/itwinjs-core/blob/1e97a4f18e1c7339863c5bd916b8e75a7dd39422/core/common/src/rpc/web/BentleyCloudRpcProtocol.ts#L55